content/rdi/quickstart/write-behind-guide.md

```diff
@@ -39,16 +41,21 @@ RDI write-behind currently supports these target data stores:
 | Redis Enterprise     |
 | SQL Server |
 
-
 ## Prerequisites
 
 The only prerequisite for running RDI write-behind is [Redis Gears Python](https://redis.com/modules/redis-gears/) >= 1.2.6 installed on the Redis Enterprise Cluster and enabled for the database you want to mirror to the downstream data store.
 For more information, see [Redis Gears installation]({{<relref "/rdi/installation/install-redis-gears">}}).
 
-## Preparing the write-behind pipeline
+## Preparing the write-behind & read-through pipeline
 
 - Install [RDI CLI]({{<relref "/rdi/installation/install-rdi-cli">}}) on a Linux host that has connectivity to your Redis Enterprise Cluster.
 - Run the [`configure`]({{<relref "/rdi/reference/cli/redis-di-configure">}}) command to install the RDI Engine on your Redis database, if you have not used this Redis database with RDI write-behind before.
+- **For Read Through only** the Redis database proxy has to be reconfigured using the [rladmin CLI](https://docs.redis.com/latest/rs/references/cli-utilities/rladmin/):
+
+  ```bash
+  rladmin> tune db redis-di-1 schedpolicy mnp
```

Comment:
redis-di-1 is normally how this BDB should be named, but it can be something else (redis-di-2, etc.).

Reply:
done